src/order/complete_boolean_algebra.lean

```diff
@@ -2,11 +2,23 @@
 Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl
-
-Theory of complete Boolean algebras.
 -/
 import order.complete_lattice
 
+/-!
+# Completely distributive lattices and boolean algebras
```

Comment:
Shouldn't this be
```suggestion
# Complete distributive lattices and boolean algebras
```
My understanding was that "complete" modifies "lattice" not "distributive", but I'm not formally trained in this area.

Reply:
Yeah, but here the "completely" does apply to "distributive". It says that sup and inf distribute over the complete operations, that is Sup and Inf. The fully clear name for it would be "completely distributive complete lattice". "complete distributive lattice" would simply mean "complete lattice" as completeness implies distributivity.

Reply:
Ah, the crucial comment below was hidden behind a fold:
```lean
/-- A complete distributive lattice is a bit stronger than the name might
  suggest; perhaps completely distributive lattice is more descriptive,
```

Reply:
```suggestion
# Completely distributive lattices and Boolean algebras
```

Reply:
I don't really like this upper casing. For example, we're not writing "Abelian group" (although we say "Noetherian ring"?)

Reply:
Yes, "abelian" is a bit of [an exception](https://en.wikipedia.org/wiki/Abelian_group#A_note_on_the_typography). I think it's safest to follow Wikipedia which capitalizes the "Boolean" in "Boolean algebra": https://en.wikipedia.org/wiki/Boolean_algebra_(structure)